scripts/collisions.js

```diff
@@ -87,12 +104,14 @@ const worker = (methodName, done) => {
     methodName,
     mean: stats.amean().toFixed(2),
     stddev: stats.stddev().toFixed(2),
-    moe: (stats.moe() / stats.amean()).toFixed(2),
-    runs: stats.length,
+    moe: computeMoe().toFixed(2),
+    runs: numRuns,
+    n: stats.length,
```

Comment:
`n` and `runs` would differ in cases where no collisions were found: https://github.com/snaplet/copycat/pull/32/files#r994766064

* `n` is the number of collision data set size datapoints
* `runs` is the total number of runs - including the runs where there were no collisions and `MAX_N` was reached